src/components/Form/ServiceForm.tsx

```diff
@@ -107,17 +110,37 @@ function ServiceForm() {
         // Get platform signature
         const signature = await getServiceSignature({ profileId: Number(user?.id), cid });
 
-        const contract = new ethers.Contract(
-          config.contracts.serviceRegistry,
-          ServiceRegistry.abi,
-          signer,
-        );
-        const tx = await contract.createService(
-          user?.id,
-          process.env.NEXT_PUBLIC_PLATFORM_ID,
-          cid,
-          signature,
-        );
+        const getUser = await getUserByAddress(user.address);
+        const delegateAddresses = getUser.data?.data?.users[0].delegates;
+
+        let tx;
+        if (
+          delegateAddresses &&
```

Comment:
need to have the env boolean here, like that the platform can decide at any time to stop it

Reply:
and need to be the first check, better for perf to do the simplest one first